### PR TITLE
TECH-1108: Add and configure cypress integration tests

### PR DESCRIPTION
**Pull request by jahia-carp**
https://jira.jahia.org/browse/TECH-1108

### CODE REVIEW

**Review by Fgerthoffert — CHANGES_REQUESTED** [review]
The nightly and manual-run workflows seems to be missing

File: .github/workflows/on-code-change.yml [hunk 1]
```diff
@@ -58,3 +58,62 @@ jobs:
...
+          node-version: 'lts/*'
+      - uses: jahia/jahia-modules-action/integration-tests@v2
+        with:
+          module_id: siteSettings
+          testrail_project: Site Settings Module
+          jahia_artifact_name: Integration-Tests-Standalone
+          tests_manifest: provisioning-manifest-build.yml
+          jahia_image: jahia/jahia-discovery-dev:8-SNAPSHOT
+          should_use_build_artifacts: true
+          github_artifact_name: ctm-tests-artifacts-${{ github.run_number }}
```
[the remark this answers is not in the document]
Comment (jahia-carp) [reply .github/workflows/on-code-change.yml:92]: Fixed

File: .github/workflows/on-merge.yml [hunk 2]
```diff
@@ -39,9 +39,68 @@ jobs:
...
+          node-version: 'lts/*'
+      - uses: jahia/jahia-modules-action/integration-tests@v2
+        with:
+          module_id: siteSettings
+          testrail_project: Site Settings Module
+          jahia_artifact_name: Integration-Tests-Standalone
+          tests_manifest: provisioning-manifest-build.yml
+          jahia_image: jahia/jahia-discovery-dev:8-SNAPSHOT
+          should_use_build_artifacts: true
+          github_artifact_name: ctm-tests-artifacts-${{ github.run_number }}
```
[the remark this answers is not in the document]
Comment (jahia-carp) [reply .github/workflows/on-merge.yml:72]: Fixed

File: tests/cypress/e2e/basic.test.cy.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+describe('First cypress lunch', () => {
```
[the remark this answers is not in the document]
Comment (jahia-carp) [reply tests/cypress/e2e/basic.test.cy.ts:1]: Fixed :-)

File: tests/maven.settings.xml [hunk 4]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
```
[the remark this answers is not in the document]
Comment (jahia-carp) [reply tests/maven.settings.xml:1]: Wrong, removed

File: .github/workflows/on-code-change.yml [hunk 5]
```diff
@@ -58,3 +58,62 @@ jobs:
...
+      - uses: KengoTODA/actions-setup-docker-compose@main
+        with:
+          version: '1.29.2'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+      - uses: jahia/jahia-modules-action/integration-tests@v2
+        with:
+          module_id: siteSettings
+          testrail_project: Site Settings Module
```
[the remark this answers is not in the document]
Comment (jahia-carp) [reply .github/workflows/on-code-change.yml:87]: Thanks

**Review by Fgerthoffert — DISMISSED** [review]
Ok for me for the CI/CD part, on the assumption all is passing (build, tests...)

**Review by Fgerthoffert — APPROVED** [review]

**Pull request merged**